src/altinn-app-frontend/__mocks__/initialStateMock.ts

```diff
@@ -78,19 +80,14 @@ export function getInitialStateMock(customStates?: Partial<IRuntimeState>): IRun
           logo: '',
           orgnr: '',
           homepage: '',
-          environments: [
-            'tt02',
-            'production',
-          ],
+          environments: ['tt02', 'production'],
```

Comment:
Is this file changed or just re-formatted?

Reply:
It is changed. On line 130 we added the appLanguagestate to the mock